tests/GovUk.Frontend.AspNetCore.Tests/TagHelpers/ErrorSummaryTagHelperTests.cs

```diff
@@ -77,6 +77,52 @@ public async Task ProcessAsync_GeneratesExpectedOutput()
         AssertEx.HtmlEqual(expectedHtml, output.ToHtmlString());
     }
 
+    [Fact]
+    public async Task ProcessAsync_Empty_GeneratesExpectedOutput()
+    {
+        // Arrange
+        var context = new TagHelperContext(
+            tagName: "govuk-error-summary",
+            allAttributes: new TagHelperAttributeList(),
+            items: new Dictionary<object, object>(),
+            uniqueId: "test");
+
+        var output = new TagHelperOutput(
+            "govuk-error-summary",
+            attributes: new TagHelperAttributeList(),
+            getChildContentAsync: (useCachedResult, encoder) =>
+            {
+                var errorSummaryContext = (ErrorSummaryContext)context.Items[typeof(ErrorSummaryContext)];
+
+                errorSummaryContext.SetTitle(new AttributeDictionary(), new HtmlString("Title"));
+                errorSummaryContext.SetDescription(new AttributeDictionary(), new HtmlString("Description"));
+
+                var tagHelperContent = new DefaultTagHelperContent();
+                return Task.FromResult<TagHelperContent>(tagHelperContent);
+            });
+
+        var tagHelper = new ErrorSummaryTagHelper()
+        {
+            ViewContext = new ViewContext()
+        };
+
+        // Act
+        await tagHelper.ProcessAsync(context, output);
+
+        // Assert
+        var expectedHtml = @"
+<div class=""govuk-error-summary"" data-module=""govuk-error-summary"">
+    <div role=""alert"">
+        <h2 class=""govuk-error-summary__title"">Title</h2>
+        <div class=""govuk-error-summary__body"">
+            <p>Description</p>
+        </div>
+    </div>
+</div>";
+
+        AssertEx.HtmlEqual(expectedHtml, output.ToHtmlString());
+    }
+
```

Comment:
Extra test case to verify behaviour when there are zero error items.

Seems to be introduced in v5.4.0, where the `<ul>` only gets created if the item count is truthy/non-zero:

https://github.com/alphagov/govuk-frontend/pull/4971/files#diff-43521f7999f31d5b19bb203342ab8c5e5ff212b456e0fc072bca7656575e5f6a

![image](https://github.com/user-attachments/assets/d6ec4c42-28e4-487d-b1c6-31791422e7d2)